media/css/firefox/election.scss

```diff
@@ -18,6 +18,13 @@ $image-path: '/media/protocol/img';
     padding-bottom: 200px;
 }
 
+.mzp-c-hero-title {
+    @include at2x('/media/img/firefox/election/firefox-logo.png', 100px);
```

Comment:
Suggest adding this to `/img/logos/firefox/` as a central place, so we don't end up with multiple copies of the same wordmark logo?

Reply:
The same image only twice the size is used on the WN67 page. The size on 67 seems more likely to be the one used centrally so I moved that one. Which means I removed the `a2x` mixin here and just use the low-res version at half its size.